custom_components/sector/entity.py

```diff
@@ -33,9 +33,9 @@ def __init__(
         self._model = model
         _LOGGER.debug(
             "Initialized entity %s %s with unique_id: %s",
+            self._serial_no,
```

Comment:
The message would now be `Initialized entity [serial] [name] with unique_id: [model]`
I guess the two first is ok but model is not the unique id